browser/ui/brave_browser_command_controller.cc

```diff
@@ -146,8 +146,10 @@ void BraveBrowserCommandController::UpdateCommandForWebcompatReporter() {
 #if BUILDFLAG(ENABLE_TOR)
 void BraveBrowserCommandController::UpdateCommandForTor() {
   const bool is_tor_enabled = !tor::TorProfileService::IsTorDisabled();
+  const bool is_guest_session = browser_->profile()->IsGuestSession();
   UpdateCommandEnabled(IDC_NEW_TOR_CONNECTION_FOR_SITE, is_tor_enabled);
-  UpdateCommandEnabled(IDC_NEW_OFFTHERECORD_WINDOW_TOR, is_tor_enabled);
+  UpdateCommandEnabled(IDC_NEW_OFFTHERECORD_WINDOW_TOR,
```

Comment:
I think guest window also doesn't need `IDC_NEW_TOR_CONNECTION_FOR_SITE` command.

Reply:
Hm yes, but we currently only insert that command for Tor profile though.

Reply:
I see. it seems there is no way to trigger that command except app menu.